### ISSUE TITLE
Prism3 - Availability incorrect

### ISSUE THREAD

**Issue by cpwood** [LibrariesHacked/catalogues-library#5]
Hi - just discovered this amazing library!

I've noticed that the stock availability isn't working fully for Prism3 - at least not here in Wigan!

The HREF that's being searched for is `http://schema.org/InStock` (see [here](https://github.com/LibrariesHacked/catalogues-library/blob/7ac6e432a55a75e8aa05518976b0ab7452eafbaf/connectors/prism3.js#L79)) however it should also include `http://schema.org/InStoreOnly`.

<img width="1230" alt="CleanShot 2021-12-04 at 23 01 53@2x" src="https://user-images.githubusercontent.com/13966104/144727208-be373a28-8416-4d61-9750-b1e769087707.png">

https://capitadiscovery.co.uk/wigan/items/486163?query=bob+mortimer&resultsUri=items%3Fquery%3Dbob%2Bmortimer

**Comment by cpwood** [comment]
Just as a PS, I wanted to thank you again for this wonderful npm library. I've used it on a personal project in my free time over the course of the last week to build Libraree.org (with a temporary hack to deal with the above issue, for now).

It was something to solve a personal need: searching a short list of several authorities' catalogues in one go; something I've been doing the long way for a while now. Your hard work gave me the nudge to build the app I'd been dreaming about for a while!

It differs from librarybooks.uk in so much as it searches a predefined short list of libraries of which I'm already a member, rather than all library services in the UK by distance. It also uses Google Books to turn titles into ISBN numbers. 

You can install it as an app on phones or on a desktop computer as it's also a PWA. 

I realise a comment on an issue isn't the best way of letting you know all this, but I've struggled to find another way of reaching you. 

Thanks again!

**Comment by DaveBathnes** [comment]
Hi Chris, 

Thanks for raising this, and for the feedback! Availability is definitely one of the biggest challenges I've found with this project as it often seems to be down to the configuration within each individual library service. I've just made a change that should add that additional status in though.

It's also good to hear about Libraree - just taken a look and it looks great, I'll put it on my phone!
